R/QuestionTab.R

```diff
@@ -36,13 +61,20 @@ QuestionTab <- R6Class(
       self$exposure <- exposure
       self$type <- type
       self$subtype <- subtype
+      self$ui_pars <- ui_pars
```

Comment:
What does pars mean?  Can we have a more useful word for this variables: settings, pairs etc

Reply:
renamed to "settings"